src/SFA.DAS.Roatp.CourseManagement.Jobs/Infrastructure/Firewall/ApiError.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Newtonsoft.Json;
+
+namespace SFA.DAS.Roatp.CourseManagement.Jobs.Infrastructure.Firewall
```

Comment:
Do we need our internal apis to do this, return ApiError in case of unsuccessful request?

Reply:
I lifted this as part of lift from das-roatp-functions, and saw it had a comment about being used by the WAF, but now I check it, doesn't seem used.,  So removing it.